build.sbt

```diff
@@ -17,6 +17,9 @@ def formattingTestArg(target: File) = Tests.Argument("-u", target.getAbsolutePat
 credentials ++= loadM2Credentials(streams.value.log)
 resolvers ++= loadM2Resolvers(sLog.value)
 
+lazy val myRepo = "My Repo" at "https://s3.us-east-2.amazonaws.com/ichernetsky/maven/"
```

Comment:
why do we need this? was this commited by mistake?

Reply:
is it because of mesos JVM lib? Can we upload it to the mesosphere maven repo as we did for the special ipv6 build... ?

Reply:
Yes, it is. Once it is available from the Mesos repo, I will remove this.